### ISSUE TITLE
Error on training

### ISSUE THREAD

**Issue by jpswensen** [prosodylab/prosodylab.dictionaries#2]
I have tried to muddle my way through how to do a new training. The problem is that if I do something like 

    python3 -m aligner -c eng.yaml -d eng.dict -t new_data -w output -V

I get an error from HERest about missing a vocabulary list.

    Preparing corpus 'new_data'.
    Preparing aligner.
    Training aligner on corpus 'new_data'.
    Flat start training.
    Training iteration 1.
      ERROR [+2319]  HERest: file name of vocabulary list expected
     FATAL ERROR - Terminating program HERest

I assume this means I am missing a file either in the training directory, that HTK got installed funny on my Mac, or some other problem. Have you ever seen this? I went the normal routes of looking on StackOverflow and Google before posting this.

**Comment by kylebgorman** [comment]
No, I've never seen that before. It does appear to indicate that HERest is
having trouble parsing shell arguments. I wonder if your file names have
spaces in them or something like that?

If you want to send me a reproducible example via email, I can take a
closer look.
